### PR TITLE
chore: make the project contributor friendly

### PR DESCRIPTION
**Pull request by orama254**
enforce [community standards](https://github.com/droidconKE/droidconKE2022ReactNative/community) by making the project contributor friendly.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
+# Contributing to DroidconKe-RN App
+
+
+You can contribute to Hey Linda by beta testing, recording meditations, or submitting code.
```
Comment (brianwachira) [inline CONTRIBUTING.md]: I guess this line should be corrected to something else?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+
+
+1. Fork this repository to your own repositiry. 
+
+2. Clone the forked repositiry to your local machine.
+
+3. Run it locally with our [guide](#how-to-run-locally)
+
+4. Create your feature branch: `git checkout -b my-new-feature`
```
Comment (brianwachira) [inline CONTRIBUTING.md]: I am thinking the feature branch name could be named as
`git checkout -b feature-my-new-feature`

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+5. make changes to the project.
+
+6. Commit your changes: `git commit -m 'Add some feature'`
+
+7. Push to the branch: `git push origin my-new-feature`
```
Comment (brianwachira) [inline CONTRIBUTING.md]: I am thinking the feature branch name could be named as
`git checkout -b feature-my-new-feature`

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+<br>
+
+## Contribution steps
+
+
+
+
+1. Fork this repository to your own repositiry. 
+
+2. Clone the forked repositiry to your local machine.
```
Comment (makunomark) [inline CONTRIBUTING.md]:
```suggestion
2. Clone the forked repository to your local machine.
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+
+3. Run it locally with our [guide](#how-to-run-locally)
+
+4. Create your feature branch: `git checkout -b my-new-feature`
+
+5. make changes to the project.
```
Comment (makunomark) [inline CONTRIBUTING.md]:
```suggestion
5. Make changes to the project.
```

File: CONTRIBUTING.md (hunk 1 above)
Comment (orama254) [reply CONTRIBUTING.md]: totally missed this, rectifying asap

**Review by makunomark — APPROVED** [review]

**Review by brianwachira — APPROVED** [review]

**Pull request merged**